R/data.R

```diff
@@ -33,3 +33,32 @@
 #'   June 7 is the average of the underlying data for June 1 through 7,
 #'   inclusive.
 "case_death_rate_subset"
+
```

Comment:
I am not sure if this dataset should be a "built-in" dataset for the package, since it's not epi data. Would it be better for it to go in `inst/extdata`?

Reply:
I don't think there's any harm in leaving it built in as-is.